backend/Dockerfile

```diff
@@ -32,7 +35,7 @@ RUN PACKAGES="make subversion ca-certificates gcc git" \
 		*.dcl *.icl \
 		Dockerfile \
 		Makefile \
-	&& rm -rf /opt/clean \
+	&& rm -rf /opt/clean/* \
```

Comment:
It's not only that, `rm *` takes more CPU cycles because you need to read the directory index; deleting a single directory removes it from the parent in constant time.